swaybar/ipc.c

```diff
@@ -153,7 +153,7 @@ static bool ipc_parse_config(
 		return false;
 	}
 	json_object *markup, *mode, *hidden_state, *position, *status_command;
-	json_object *font, *bar_height, *wrap_scroll, *workspace_buttons, *strip_workspace_numbers;
+	json_object *font, *bar_height, *wrap_scroll, *workspace_buttons, *strip_workspace_numbers, *strip_workspace_name;
```

Comment:
This whole function appears to ignore the 80 character line limit, but that doesn't mean you should do the same. I'm not fussed as to whether you wrap the other lines or not.